django_airavata/apps/groups/views.py

```diff
@@ -2,22 +2,174 @@
 from django.conf import settings
 from django.contrib.auth.decorators import login_required
 from django.shortcuts import render, redirect
+from .forms import CreateForm, AddForm, RemoveForm
+from django.contrib import messages
+from apache.airavata.model.sharing.ttypes import UserGroup
+from apache.airavata.model.sharing.ttypes import GroupCardinality
+import datetime
+import uuid
 
 import logging
 
 logger = logging.getLogger(__name__)
 
+@login_required
+def groups_manage(request):
+
+    gateway_id = settings.GATEWAY_ID
+    username = request.user.username
+
+    try:
+        owner_list = request.sharing_client.getGroups(gateway_id, 0, -1)
```

Comment:
Don't we want to filter out the user's single user group? That is, this group management UI should only manage multi-user groups so I think we should always filter out single user groups.